aio/deploy/helm-chart/kubernetes-dashboard/README.md

```diff
@@ -100,6 +100,11 @@ Parameter                                       | Description
 `ingress.customPaths`                           | Override the default ingress paths                                                                                               | `[]`
 `ingress.hosts`                                 | Dashboard Hostnames                                                                                                              | `nil`
 `ingress.tls`                                   | Ingress TLS configuration                                                                                                        | `[]`
+`dashboardSettings.clusterName`                 | Cluster name that appears in the browser window title if it is set | `""`
```

Comment:
I'd rename it to `settings.xxx`.

Reply:
`settings`, being a dict with default values as `{}` ?

Reply:
Used an empty dict and added example in the values.yaml